presto-hive/src/main/java/com/facebook/presto/hive/HiveConnector.java

```diff
@@ -64,6 +65,7 @@
 
     private final ConnectorAccessControl accessControl;
     private final ClassLoader classLoader;
+    private final ConnectorPlanOptimizerProvider connectorPlanOptimizerProvider;
```

Comment:
let's call this `planOptimizerProvider` given it's already in `HiveConnector` class